src/Compilers/Test/Core/AssemblyLoadTestFixture.cs

```diff
@@ -189,6 +194,36 @@ public void Write(StringBuilder sb, string s)
             var delta1Reference = MetadataReference.CreateFromFile(Delta1);
             DeltaPublicSigned1 = GenerateDll("DeltaPublicSigned", _directory.CreateDirectory("Delta1PublicSigned"), Delta1Source, publicKeyOpt: SigningTestHelpers.PublicKey);
 
+            const string Delta2Source = @"
+using System.Text;
+
+[assembly: System.Reflection.AssemblyTitle(""Delta"")]
+[assembly: System.Reflection.AssemblyVersion(""2.0.0.0"")]
+
+namespace Delta
+{
+    public class D
+    {
+        public void Write(StringBuilder sb, string s)
+        {
+            sb.AppendLine(""Delta.2: "" + s);
+        }
+
+        public void M2()
```

Comment:
It looks like the only real change in here is that 'M1' and 'M2' were added, right?